### ISSUE TITLE
'use-data-uris' option should be true by default

### ISSUE THREAD

**Issue by Martin1887** [sytsereitsma/mdbook-plantuml#59]
I think that the `use-data-uris` option is a saner manner of rendering images and therefore a better default.

Thanks and Regards,

**Comment by sytsereitsma** [comment]
I agree. I turned them off by default in this first version, because I wasn;t sure about the support for data uris on all platforms.
No-one has complained so far, so I guess I can flip the logic.
